tests/unit/type/hash.tcl

```diff
@@ -762,7 +762,7 @@ start_server {tags {"hash external-ok"}} {
         assert_equal [dict get $k ZIP_STR_32B] [string repeat x 65535]
         set k [dict remove $k ZIP_STR_32B]
         set _ $k
-    } {ZIP_INT_8B 127 ZIP_INT_16B 32767 ZIP_INT_32B 2147483647 ZIP_INT_64B 9223372036854775808 ZIP_INT_IMM_MIN 0 ZIP_INT_IMM_MAX 12}
+    } {ZIP_INT_8B 127 ZIP_INT_16B 32767 ZIP_INT_32B 2147483647 ZIP_INT_64B 9223372036854775808 ZIP_INT_IMM_MIN 0 ZIP_INT_IMM_MAX 12} {needs:config-sanitize}
```

Comment:
Ignoring a config error makes most sense to me here.

Reply:
I think ignoring config errors here makes most sense.

Reply:
ok, so maybe instead of `r config set ....` we'll add some `proc config_set` that can ignore it in some cases (e.g. on external), but not in others?
i.e. i don't want to just add a `catch` in that test explicitly, and unconditionally.